README.md

```diff
@@ -1076,6 +1076,10 @@ The use of Amazon Voice Focus via this SDK involves the downloading and executio
 
 The use of Amazon Voice Focus runtime code is subject to additional notices. See [this NOTICES file](https://static.sdkassets.chime.aws/workers/NOTICES.txt) for details. You agree to make these additional notices available to all end users who use Amazon Voice Focus runtime code via this SDK.
 
+The browser demo applications in the demos directory (https://github.com/aws/amazon-chime-sdk-js/tree/master/demos) use TensorFlow.js (https://github.com/tensorflow/tfjs) and pre-trained TensorFlow.js models (https://github.com/tensorflow/tfjs-models) for image segmentation. Use of these third party models involves downloading and execution of code at runtime from jsDelivr (https://www.jsdelivr.com/) by end user browsers. For the jsDelivr Acceptable Use Policy, please visit this link (https://www.jsdelivr.com/terms/acceptable-use-policy-jsdelivr-net).
```

Comment:
I think you want to link the browser demo here right?
Also for consistency we could markdown the links.
The [browser demo application](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/browser) uses [TensorFlow.js](https://github.com/tensorflow/tfjs) and pre-trained [TensorFlow.js models](https://github.com/tensorflow/tfjs-models) for image segmentation. Use of these third party models involves downloading and execution of code at runtime from [jsDelivr](https://www.jsdelivr.com/) by end user browsers. For the jsDelivr Acceptable Use Policy, please visit [this link](https://www.jsdelivr.com/terms/acceptable-use-policy-jsdelivr-net).

Reply:
By all means make sure the text is linkified, but keep the hyperlinks readable. This text might be copied and pasted in plain text.

Reply:
fixed.